src/main/java/Calculator.java

```diff
@@ -93,13 +70,11 @@ private static boolean checkOps(String text) {
 
     public static ArrayList<Integer> filterValue(String[] inputList){
         ArrayList<Integer> valueList = new ArrayList<Integer>();
-
         IntStream.range(0, inputList.length).forEach(idx -> {
-            if(idx%2==0){
+            if (idx%2 == 0){
```

Comment:
포맷팅 관련 코멘트를 많이 드리는데요
idea의 reformat code 기능을 활용하시는걸 적극 추천드립니다
```suggestion
            if (idx % 2 == 0) {
```